src/multiint.cpp

```diff
@@ -3407,6 +3444,14 @@ static SwapPlayerIndexesResult recvSwapPlayerIndexes(NETQUEUE queue, const std::
 				}
 			}
 		}
+
+		if ((game.blindMode != BLIND_MODE::NONE) && (selectedPlayer < MAX_PLAYERS))
+		{
+			std::string blindLobbyMessage = _("BLIND LOBBY NOTICE:");
+			blindLobbyMessage += "\n";
```

Comment:
Let's include `- ` in this part to align message construction with the `printBlindModeHelpMessagesToConsole()` function